### PR TITLE
include elm format as dev tool

### PR DESCRIPTION
**Pull request by mburri**
References #31

### CODE REVIEW

File: elm.mk [hunk 1]
```diff
@@ -57,11 +59,13 @@ MO_URL := "https://raw.githubusercontent.com/tests-always-included/mo/${MO_VERSI
 ifeq ($(OS),Darwin)
 	DEVD_URL := "https://github.com/cortesi/devd/releases/download/v${DEVD_VERSION}/devd-${DEVD_VERSION}-osx64.tgz"
 	ELM_URL := "https://github.com/elm/compiler/releases/download/${ELM_VERSION}/binaries-for-mac.tar.gz"
+	ELM_FORMAT_URL: = "https://github.com/avh4/elm-format/releases/download/${ELM_FORMAT_VERSION}/elm-format-${ELM_FORMAT_VERSION}-mac-x64.tgz"
```
Comment (cloud8421) [inline elm.mk]: I believe the `:` is misplaced, this should read:

```
ELM_FORMAT_URL := "https://github.com/avh4/elm-format/releases/download/${ELM_FORMAT_VERSION}/elm-format-${ELM_FORMAT_VERSION}-mac-x64.tgz"
```

**Review by cloud8421 — CHANGES_REQUESTED** [review]
Thanks for this! Happy to have it as a feature. Please see my comment about a typo that causes the build to fail.

Can I please ask you to add a test for the new file in https://github.com/cloud8421/elm.mk/blob/master/test.rb?

File: elm.mk (hunk 1 above)
Comment (mburri) [reply elm.mk]: uups... working on linux :-)
have to check the tests

**Comment by mburri** [comment]
I'm not sure why the travis build fails now - Works for my fork: https://travis-ci.org/mburri/elm.mk/
Am I missing something here?

**Comment by cloud8421** [comment]
@mburri seems related to a custom parallelisation strategy I used before elm 0.19 (CI only). I reckon it can be removed.

EDIT: made some simplification in #33 and it's stable now.

**Review by cloud8421 — APPROVED** [review]
Well done, thanks!

**Comment by cloud8421** [comment]
@mburri please feel free to merge anytime (so that your authorship is preserved). Once again thanks for taking the time to work on this.

**Comment by mburri** [comment]
I merged your branch and now everything looks fine! thank you!

**Comment by cloud8421** [comment]
@mburri glad to hear! See https://github.com/cloud8421/elm.mk/pull/32#issuecomment-498575208 for the next step :)

**Pull request merged**